Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -27,38 +27,37 @@ public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListene
         /// <returns>
         /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
         /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
+        public static IDisposable Enable(TelemetryConfiguration configuration = null, IEnumerable<string> excludedDomains = null)
```

Comment:
No, you're referring to the DependencyTrackingTelemetryModule class, and that is only used in the .NET Framework scenario. This is specifically for the .NET Core scenario. In this case, a user has to explicitly call DependencyCollectorDiagnosticListener.Enable() to enable the dependency collector in their .NET Core application.